### PR TITLE
IA-660: add missing translations based on script

### PR DESCRIPTION
**Pull request by quang-le**
## Changes
- Added translations for missing keys
- Updated keys and translations in `AddTaskComponent`, `ImportGeoPckgDialog` and `VersionsDialog` to match existing conventions and avoid ambiguity
- Added translations for user permissions (only those currently available in Iaso, not all those available in admin)
- Updated and corrected some translations

### CODE REVIEW

File: hat/assets/js/apps/Iaso/domains/dataSources/components/ImportGeoPkgDialog.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ const ImportGeoPkgDialog = ({
 
     const titleMessage = versionNumber ? (
         <FormattedMessage
-            id="update_from_gpkg"
+            id="iaso.geopPckg.label.updateFromGpkg"
```
Comment (olethanh) [inline hat/assets/js/apps/Iaso/domains/dataSources/components/ImportGeoPkgDialog.js:66]: GeoPkg

**Comment by olethanh** [comment]
```      defaultMessage: 'Updated at',
        defaultMessage: 'Updated',```
In some other table created_at and updated_at column is translated as  Time Created and Time Updated, we should standardise that
```    timeCreated: {
        defaultMessage: 'Time Created',
        id: 'iaso.devices.timeCreated',
    },
    timeUpdated: {
        defaultMessage: 'Time Updated',
        id: 'iaso.devices.timeUpdated',
    },
```

File: hat/assets/js/apps/Iaso/domains/users/messages.js [hunk 2]
```diff
@@ -101,6 +101,37 @@ const MESSAGES = defineMessages({
         id: 'iaso.users.dialog.chooseLocation',
         defaultMessage: 'Select location(s)',
     },
+    iaso_completeness: {
+        id: 'iaso.permissions.completeness',
```
Comment (olethanh) [inline hat/assets/js/apps/Iaso/domains/users/messages.js:105]: should this not be in English?

File: hat/assets/js/apps/Iaso/domains/app/translations/fr.json [hunk 3]
```diff
@@ -1,9 +1,12 @@
 {
     "blsq.treeview.label.selectMultiple": "Sélectionner Org Unit(s)",
     "blsq.treeview.label.selectSingle": "Sélectionner une Org Unit",
+    "blsq.treeview.loading": "Chargement",
     "blsq.treeview.search.cancel": "Annuler",
     "blsq.treeview.search.confirm": "Confirmer",
+    "blsq.treeview.search.inputLabelObject": "Rechercher",
     "blsq.treeview.search.options.label.clear": "Effacer",
+    "blsq.treeview.search.options.label.noOptions": "Auncun résultat trouvé",
```
Comment (olethanh) [inline hat/assets/js/apps/Iaso/domains/app/translations/fr.json:9]: aucun

File: hat/assets/js/apps/Iaso/domains/app/translations/fr.json [hunk 4]
```diff
@@ -481,14 +513,21 @@
     "iaso.snackBar.updateDefaultSourceError": "Une erreur est survenue en sauvant la source par défaut",
     "iaso.snackBar.updateFormError": "Une erreur est survenue en sauvant le formulaire",
     "iaso.snackBar.updateFormVersionError": "Une erreur est survenue en mettant à jour une version de formulaire",
+    "iaso.sourceVersion.label.create": "Créer une nouvelle version depuis DHIS2",
+    "iaso.sourceVersion.label.create_explication": "Importer les OrgUnits depuis un serveur DHIS2",
+    "iaso.sourceVersion.label.import_task_explication": "L'import va se dérouler en arrière-plan et peut prendre une dizaine de minutes",
+    "iaso.sourceVersion.label.update": "Mise à jour de la version {version}",
+    "iaso.sourceVersion.label.update_explication": "Mettre à jour cette version en la synchronsant avec DHIS2. Les nouvelles OrgUnits seront importées depuis DHIS2, mais les OrgUnits existantes sur cette version ne seront pas modifiées.",
```
Comment (olethanh) [inline hat/assets/js/apps/Iaso/domains/app/translations/fr.json:520]: synchronisant

File: hat/assets/js/apps/Iaso/domains/app/translations/fr.json [hunk 5]
```diff
@@ -517,6 +556,13 @@
     "iaso.users.permissions": "Permissions",
     "iaso.users.selectedOrgUnits": "Unité d'organisation sélectionnées",
     "iaso.users.update": "Mettre l'utilisateur à jour",
+    "iaso.versionsDialog.label.createdAt": "Création",
+    "iaso.versionsDialog.label.datasource.noVersion": "Cette source est vide, veuillez créer une version avec l'un des boutons ci-dessous",
+    "iaso.versionsDialog.label.description": "Description",
+    "iaso.versionsDialog.label.dialogVersions": "Versions pour la source: {source}",
+    "iaso.versionsDialog.label.newVersionDhis2": "Nouvelle version depuis DHIS2",
+    "iaso.versionsDialog.label.newVersionGpkg": "Nouvelle version sur base d'un Geopackage",
+    "iaso.versionsDialog.label.number": "Nombre",
```
Comment (olethanh) [inline hat/assets/js/apps/Iaso/domains/app/translations/fr.json:565]: numéro

**Review by olethanh — COMMENTED** [review]
as a reminder for my self i will add the correction

**Review by beygorghor — COMMENTED** [review]
I still have one missing translation in the console
Screenshot 2021-09-10 at 14 54 34

**Comment by olethanh** [comment]
good catch it come from another PR, added it

**Pull request merged**